### PR TITLE
remove datadog agents

### PR DESCRIPTION
**Pull request by syed-awais-ali**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

**Review by feanil — COMMENTED** [review]
This should probably not go into the common role since we don't want to affect all users.  Why not just create a new small playbook with the things we need.

File: playbooks/roles/common_vars/defaults/main.yml [hunk 1]
```diff
@@ -225,3 +225,5 @@ COMMON_JWT_SECRET_KEY: 'SET-ME-PLEASE'
 CREATE_SERVICE_WORKER_USERS: True
 
 COMMON_ENABLE_AWS_ROLE: true
+
+COMMON_DISABLE_DATADOG: false
```
Comment (feanil) [inline playbooks/roles/common_vars/defaults/main.yml]: I don't think you need this variable.  It's not used anywhere.

File: playbooks/roles/datadog-uninstall/meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - common
```
Comment (feanil) [inline playbooks/roles/datadog-uninstall/meta/main.yml]: I don't think you need this either.  It doesn't need common to uninstall datadog.

File: playbooks/roles/datadog-uninstall/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+common_datadog_agent_version: '1:5.10.1-1'
```
Comment (feanil) [inline playbooks/roles/datadog-uninstall/defaults/main.yml]: These variables should not be prefixed with common,  they are now in a different role.  prefix with `datadog_uninstall`

**Review by coryleeio — APPROVED** [review]

File: playbooks/roles/datadog-uninstall/defaults/main.yml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+datadog_uninstall_agent_version: '1:5.10.1-1'
```
Comment (coryleeio) [inline playbooks/roles/datadog-uninstall/defaults/main.yml:1]: If we specify the version and are wrong we might miss some servers, maybe consider removing it?

Comment (coryleeio) [reply playbooks/roles/datadog-uninstall/defaults/main.yml:1]: Approving because I dont this is worth blocking on

**Pull request merged**